src/components/apps/Notes/Notes.svelte

```diff
@@ -0,0 +1,145 @@
+<script>
+  import { theme } from '🍎/stores/theme.store';
+	import {addNote, notesStore} from './stores';
+	
+	const NEW_NOTE = {title: '', text: ''};
+	
+	let editing = false;
+	let note;
+	let selectedId;
+	let textInput;
+	let titleInput;
+	
+	$: sortedNotes = Object.values($notesStore).sort();
+	$: note = $notesStore[selectedId] || NEW_NOTE;
+	
+	function deleteNote() {
+		if (confirm('Are you sure you want to delete this note?')) {
```

Comment:
Hmm... Can you try integrating `SystemDialog` component here? Look at its usage in the rest of the project. Using browser prompts will break the feeling of operating system

Reply:
But when I add the SystemDialog component I get this error:

![Screenshot_select-area_20221228110813](https://user-images.githubusercontent.com/10300504/209766388-d8094660-7ed2-4df9-95da-1bf10b86ecba.png)

For using the system dialog component I used this file as a reference:

![Screenshot_select-area_20221228110839](https://user-images.githubusercontent.com/10300504/209766516-55760218-f350-4562-bbc5-b7f00c557a7d.png)